mesonbuild/cargo/manifest.py

```diff
@@ -441,6 +443,14 @@ class Manifest:
     features: T.Dict[str, T.List[str]] = dataclasses.field(default_factory=dict)
     target: T.Dict[str, T.Dict[str, Dependency]] = dataclasses.field(default_factory=dict)
 
+    # avoid warnings
+    workspace: T.Optional[raw.Workspace] = None
```

Comment:
Maybe instead have `IGNORE_FIELDS=['workspace', 'badges']`, and make _raw_to_dataclass() skip the warning?